test/controllers/EmbeddedCassandra.scala

```diff
@@ -0,0 +1,85 @@
+package controllers
+
+import java.io.File
+import java.util.concurrent.atomic.AtomicBoolean
+
+import org.cassandraunit.utils.EmbeddedCassandraServerHelper
+import org.slf4j.Logger
+
+import scala.concurrent.blocking
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success, Try}
+
+/**
+ * Taken from
+ *
+ * https://raw.githubusercontent.com/outworkers/phantom/develop/phantom-sbt/src/main/scala-2.10/com/outworkers/phantom/sbt/SbtPlugin.scala
+ *
+ * This solves for https://github.com/outworkers/phantom/issues/649
+ */
+object EmbeddedCassandra {
```

Comment:
This doesn't belong in here, we should probably just use the `phantom-sbt` or maybe the phantom-docker plugin or Travis CI/CCM as we do for the main build.

Reply:
The phantom-sbt plugin has the  https://github.com/outworkers/phantom/issues/649 problem where the test is buried under logging statements.

Reply:
Using phantom-docker plugin / Travis CI also makes it harder for end users to reproduce.

Reply:
@wsargent The logback problem will be investigated in more detail. I'm thinking the plugin could pre-package a standard config file, and an SBT option to point to that file on disk, so it's easy to force an override via file path.